src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -119,6 +142,7 @@ trait DataUseRestrictionSupport extends LazyLogging {
       case _ => Map.empty[AttributeName,Attribute]
     }
 
+    // this is inconsistent with our API
```

Comment:
The below section `val rsg = ...` is consistent with how we're treating gender codes for our data use api.

For ours, if RS-G is, for example, "female" the codes would be Map( "RS-G" -> true, "RS-FM" -> true, "RS-M" -> false). HERE, however, the result excludes RS-M rather than setting it as false. Which do we want to go with? If it's gender-specified, do we include the other gender that's false?

Reply:
It is the case that no value means not specifying it at all and not specifying something should mean that it does **NOT** show up as a consent code. I think the end result of the current code is correct ... you should never get a `library:consentCodes:RS-M` tag in a result set in both of the cases - no value and a false value. So, setting it to false might be nice for clarity, I don't think it's necessary for the correct results.

Reply:
From what I can tell, we _only_ use a duo of codes when calculating data use from DUOS. We use a trio of codes when building attributes for storing in ES. Furthermore, there is code that fills in `false` for missing attributes when calculating data use restriction field to index in ES... but then doesn't fill in `false` when calculating the _display_ data use restriction field for ES.

Maybe this was all intentional, but it's really difficult to be certain that there isn't a subtle bug here just from reading the code.